docs/releases/3.1.txt

```diff
@@ -303,6 +303,12 @@ Templates
   and :ttag:`blocktrans` template tags aliases continue to work, and will be
   retained for the foreseeable future.
 
+* The :ttag:`include` template tag will now also accept 'list' or 'tuple' of
+  templates name with a ``render()`` method (such as [``Template_1``, ``Template_2``])
+  as an argument. This arguments will be looked up using
+  :func:`~django.template.loader.select_template`. Such that first existing template
+  will be render.
```

Comment:
I think this can be simplified as:

```
* The :ttag:`include` template tag now uses
  :func:`~django.template.loader.select_template` so iterables of template
  names are accepted.
```

The documentation explains the feature exactly.